docker/docker-dev-debug/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '2'
+
+services:
+  metrictank:
+    security_opt:
+      - seccomp:unconfined
```

Comment:
do you have a link that best describes why we need this?
i googled, but mostly found random support tickets.

Reply:
I will add this as a comment to the docker-compose file as well
https://github.com/Microsoft/vscode-go/wiki/Debugging-Go-code-using-VS-Code#linuxdocker